src/views/pages/Splash/Splash.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+
+function LoadingMessage() {
+  return (
+    <div className="splash-screen">
+      Wait a moment while we load your app.
+      <div className="loading-dot">.</div>
+    </div>
+  );
+}
+
+function Splash(WrappedComponent) {
+  return class extends Component {
```

Comment:
I'd prefer we use function style components & `useEffect` hook instead of `componentDidMount`

Reply:
Frankly as I review this I don't think it actually needs to be a wrapped component. It's only used in one place.